nf_core/modules/module_test.py

```diff
@@ -71,25 +85,66 @@ def run(self):
     def _check_inputs(self):
         """Do more complex checks about supplied flags."""
 
+        # Retrieving installed modules
+        if self.repo_type == "modules":
+            local = False
+            installed_modules = self.module_names["modules"]
+        else:
+            local = questionary.confirm("Is the module local?", style=nf_core.utils.nfcore_question_style).unsafe_ask()
+            if local:
+                installed_modules = self.module_names.get("local/modules")
+            else:
+                installed_modules = self.module_names.get("nf-core/modules")
+
         # Get the tool name if not specified
         if self.module_name is None:
             if self.no_prompts:
                 raise UserWarning(
                     "Tool name not provided and prompts deactivated. Please provide the tool name as TOOL/SUBTOOL or TOOL."
                 )
-            modules_repo = ModulesRepo()
-            modules_repo.get_modules_file_tree()
+            if installed_modules is None:
+                raise UserWarning(
+                    "No installed modules were found from '{self.modules_repo.remote_url}'.\n"
+                    "Are you running the tests inside the nf-core/modules main directory?\n"
+                    "Otherwise, make sure that the directory structure is modules/TOOL/SUBTOOL/ and tests/modules/TOOLS/SUBTOOL/"
```

Comment:
```suggestion
                    f"No installed modules were found from '{self.modules_repo.remote_url}'.\n"
                    f"Are you running the tests inside the nf-core/modules main directory?\n"
                    f"Otherwise, make sure that the directory structure is modules/TOOL/SUBTOOL/ and tests/modules/TOOLS/SUBTOOL/"
```